packages/chrome/src/elements/nav/Nav.tsx

```diff
@@ -17,13 +18,20 @@ import { NavItemText } from './NavItemText';
 import { NavList } from './NavList';
 
 export const NavComponent = React.forwardRef<HTMLElement, INavProps>((props, ref) => {
-  const { hue, isLight, isDark } = useChromeContext();
+  const { hue, isLight } = useChromeContext();
   const navContextValue = useMemo(() => ({ isExpanded: !!props.isExpanded }), [props.isExpanded]);
 
   return (
-    <NavContext.Provider value={navContextValue}>
-      <StyledNav ref={ref} {...props} hue={hue} isLight={isLight} isDark={isDark} />
-    </NavContext.Provider>
+    <ThemeProvider
+      theme={parentTheme => ({
+        ...parentTheme,
+        colors: { ...parentTheme.colors, base: isLight ? 'light' : 'dark' }
+      })}
+    >
```

Comment:
Coerce dark vs. light mode rather than ferrying `isLight` & `isDark` props through to the various styled components.